service/history/workflow/update/registry.go

```diff
@@ -188,12 +189,18 @@ func NewRegistry(
 				withInstrumentation(&r.instrumentation),
 			)
 		} else if acc := updInfo.GetAcceptance(); acc != nil {
-			r.updates[updID] = newAccepted(
+			uc := newAccepted(
```

Comment:
nit: why "uc"?
```suggestion
			u := newAccepted(
```

Reply:
"updateAccepted" :-) Let it be just `u`.